app/src/main/java/org/mozilla/webmaker/javascript/WebAppInterface.java

```diff
@@ -290,4 +292,14 @@ public void openExternalUrl(String url) {
     public boolean isDebugBuild() {
         return BuildConfig.DEBUG;
     }
+
+    /**
+     * ----------------------------------------
+     * Get System Locale
+     * ----------------------------------------
+     */
+    @JavascriptInterface
+    public String getSystemLocale() {
+        return Locale.getDefault().toString();
```

Comment:
Did quick research and it says that `Locale.getDefault().getLanguage()` will return `English` vs `en` so I don't think we want that?